README.md

```diff
@@ -81,10 +81,6 @@ As reviewers, a PR should be only approved if following conditions are met (In o
 - git pull
 
 ## Database connection
-1. Configure the service account (in first iterations we will use a default credential)
+1. Create a folder named `keys` at the same level of the project folder.
 2. Ask for the database key and install it.
```

Comment:
how exactly is this key installed? 
maybe the second step could be just "ask for the key and save it into the 'keys' folder"

Reply:
It's the same, in there said "ask for the key and install it" the 3 step take care of saving that key inside the folder